### DISCUSSION TITLE
RustDesk v1.2.0 - black screen

### DISCUSSION THREAD

**Discussion by MrEugenX** [rustdesk/rustdesk#5243] [Q&A]
Hello. I'm experiencing black screen when opening the app on a Windows 7 PC. Why does it happen ant how to fix the problem? Thank you.

OS: Windows 7
CPU: Intel Pentium G4400
Graphics: Intel HD Graphics 510

**Comment by dinger1986** [comment]
what version are you using? Are you using sciter or flutter?

**Reply by MrEugenX** [reply]
Look at the thread name, I've posted. I've tried both v1.2.0 and v1.2.1

**Reply by dinger1986** [reply]
Yeah I changed my question, I had meant flutter or sciter version

**Reply by MrEugenX** [reply]
I don't know. It's the version downloaded from the official website.

**Reply by MrEugenX** [reply]
Can I receive support please? Why it doesn't work on a Windows 7 OS, black screen.

**Comment by ecJon** [comment]
I test it in 1.2.3.both sciter and flutter have black in win7.my client pc is win7 32bit have 2 monitor,control pc is win7 64bit,tried sciter and flutter at less 1 monitor black screen.
